src/test/java/org/jenkinsci/plugins/workflow/steps/durable_task/ShellStepTest.java

```diff
@@ -463,114 +464,10 @@ private Object writeReplace() {
         }
     }
 
-    @Ignore("TODO too flaky to run in CI")
-    @Issue("JENKINS-38381")
-    @Test public void remoteVoluminousLogger() throws Exception {
```

Comment:
Turned out to be too hard to make this be meaningful in a functional test; giving up on it for now.